Lib/test/test_string/test_templatelib.py

```diff
@@ -148,6 +148,15 @@ def test_iter(self):
         self.assertEqual(res[1].format_spec, '')
         self.assertEqual(res[2], ' yz')
 
+    def test_exhausted(self):
+        # gh-134119
+        template_iter = iter(t"{1}")
+        self.assertIsInstance(next(template_iter), Interpolation)
+        with self.assertRaises(StopIteration):
+            next(template_iter)
+        with self.assertRaises(StopIteration):
+            next(template_iter)
```

Comment:
```suggestion
        self.assertRaises(StopIteration, next, template_iter)
        self.assertRaises(StopIteration, next, template_iter)
```

This could save a bit of indentation and lines.